src/style/index.scss

```diff
@@ -148,23 +148,174 @@ $landing-color-dark-bg: $color-hack-black-5;
+    padding-top:5rem;
+  }
+  
+  .faq__flex-container{
+    display: flex;
+    justify-content: center;
+    padding-top: 75px;
+    flex-wrap: wrap;
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
+    @include media-query(814px) {
+      display: flex;
+      justify-content: center;
+      padding-top: 75px;
+      flex-wrap: wrap;
+    }
+  }
+
+  .faq__left-column{
+    width: 40rem;  
+    text-align: initial;
+    @include media-query(875px) {
+      padding-right: 1.5rem;
+    }
+  }
+
+  .faq__right-column{
+    width: 40rem;
+    text-align: initial;
+    @include media-query(875px) {
+      padding-left: 1.5rem;
+    }
+  }
+
+  .accordion__title {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 80%;
+    padding: 1rem .5rem 1rem 1rem;
+    color: $color-hack-black-80;
+      &:hover {
+        cursor: pointer;
+        color: $color-hack-red-med;
+      }
+      &.active {
+        color: $color-hack-red-med;
+      }   
+  }
+  
+  .accordion__item {
+    background-color:$color-hack-black-5;
+    box-shadow: 0px 2px 2px #E9E9E9;
+    border-radius: 4px;
+    margin-bottom: 3rem;
+    position: relative;
+
+    &:after {
+      font-weight: 900;
+      content: "+";
+      position: absolute;
+      float: right;
+      top: 1.1rem;
+      right: 1.1rem;
+      font-size: 1.5rem;
+      color: $color-hack-black-80;
+      
+    }
+    &.active {
+      &:after {
+        font-weight: 900;
+        font-size: 1.5rem;
+        content: '-';
+        color: $global-color-primary;
+      }
+    }
+    &:hover {
+      &:after {
+        cursor: pointer;
+        color: $color-hack-red-med;
+      }
+    }   
+  }
+  
+  .accordion__content {
+    opacity: 0;
+    padding: 0 1rem;
+    max-height: 0;
+    overflow: hidden;
+    clear: both;
+    opacity: 1;
+    // -webkit-transition: all 0.2s ease 0.15s;
+    // -o-transition: all 0.2s ease 0.15s;
+    transition: height 0.10s ease-in-out;
+    // active content styling
+      &.active {
+        opacity: 1;
+        // padding: 1rem;
+        max-height: 100%;
+        -o-transition: all 0.2s ease 0.15s;
+        transition: all 0.2s ease 0.15s;
+      }
+  }
+
+  .faq__wrapper {
+    position: relative;
+    
+  }
+  .faq__image-left {
+    display: none;
+
+    @include media-query($lg-up) {
+      display: initial;
+      position: absolute;
+      top: 40%;
+      left: 0;
+      width: 23vw;
+      height: auto;
+    }
+  }
+  .faq__image-right {
+    display: none;
 
+    @include media-query($lg-up) {
+      display: initial;
+      position: absolute;
+      top: 30%;
+      right: 0%;
+      width: 25vw;
+      height: auto;
+    }
+  }
+  /*--  //FAQ-SECTION  --*/
+
+  /*--  SPONSOR-SECTION  --*/
   .sponsor {
-    margin-top: calc(30vw + 12rem);
-    margin-bottom: 16rem;
+    margin-top: 50rem;
+    margin-bottom: 15rem;
   }
 
   .sponsor__heading {
     font-size: 2.8rem;
   }
 
   .sponsor-list {
+    margin-top: 
```

Comment:
missing semicolon (this is why the build's failing)

Reply:
I fixed this already you pulled a build error, sorry